src/components/HeroBanner/HeroBanner.tsx

```diff
@@ -7,7 +7,7 @@ type Props = {
 };
 const HeroBanner: React.SFC<Props> = ({ name }) => {
   return (
-    <div className={styles.container}>
+    <div className={styles.container} data-testid="Hero Banner">
```

Comment:
I rarely see people use spaces in these. They're usually namespaced like packages and are camel-cased

Reply:
yeah ill change them, idk what i was thinking here tbh because i dont do this at work neither

Reply:
Oh I'm saying it should be something more like 

```
describe('Seller Page', () => {
  describe('with succesful GET /seller request', () => {
    beforeEach(() => {
     const { findByTestId } = renderIntegration(
       '/shunfa-bakery',
       <SellerPage />
     );

     const StoryHeader = await findByTestId('Story Header');
     const OwnerPanel = await findByTestId('Owner Panel');
     const HeroBanner = await findByTestId('Hero Banner');
     const Footer = await findByTestId('Footer');
     const StoreInfo = await findByTestId('Store Info');
    });

    test('it displays display SellerPage', async () => {
      expect(StoryHeader).toBeInTheDocument();
      expect(OwnerPanel).toBeInTheDocument();
      expect(HeroBanner).toBeInTheDocument();
      expect(Footer).toBeInTheDocument();
      expect(StoreInfo).toBeInTheDocument();
    });
  });
});
```

Reply:
lol realized this comment made no sense — meant to post it here: https://github.com/sendchinatownlove/sendchinatownlove.github.io/pull/97/files#r420814499